ShadowDOM/explainer.md

```diff
@@ -572,7 +760,7 @@ The following table compares pros and cons of the various proposals:
 ## Open issues
 * What happens if a `<template shadowrootadoptedstylesheets="">` references a specifier that was imported as a non-inline CSS module whose fetch hasn’t completed yet?
 
-  Leading idea: upon creation, the shadow root would contain an empty `CSSStyleSheet`, which is the `CSSStyleSheet` for the CSS module script. When the fetch is completed and the CSS module script is fully created, this `CSSStyleSheet` is populated. This would require some changes to how module creation works in the HTML spec.
+  Leading idea: upon creation, the shadow root would contain an empty [CSS module script](https://html.spec.whatwg.org/multipage/webappapis.html#css-module-script), which is the [CSS module script](https://html.spec.whatwg.org/multipage/webappapis.html#css-module-script) for the CSS module script. When the fetch is completed and the CSS module script is fully created, this[CSS module script](https://html.spec.whatwg.org/multipage/webappapis.html#css-module-script) is populated. This would require some changes to how module creation works in the HTML spec.
```

Comment:
Another, simpler way to handle it is to just say that non-declarative imports don't work for `<template shadowrootadoptedstylesheets="">`. I think we should at least list that as an option.

Reply:
I don't love the idea of building in a dependency on fetch timing. As an example of the kinds of issues that might arise, let's say a developer is building their site on `localhost` and everything works fine because the fetches all come in before the declarative statements are reached. But then they deploy, and now in production all of their styles are broken because the fetches have actual network latency. Seems like a big pitfall.

Also, since `<template shadowrootadoptedstylesheets="">` will generally happen during the initial parse of the top-level document, I'd expect that module fetches being incomplete would be the case most of the time unless they were prefetched or cached.